src/pom.xml

```diff
@@ -1514,7 +1514,7 @@
      <printSummary>true</printSummary>
      <testFailureIgnore>${allow.test.failure.ignore}</testFailureIgnore>
      <excludedGroups>${test.excludedGroups}</excludedGroups>
-     <argLine>${argLine} -XX:+IgnoreUnrecognizedVMOptions --illegal-access=debug</argLine>
+     <argLine>@{argLine} -XX:+IgnoreUnrecognizedVMOptions --illegal-access=debug</argLine>
```

Comment:
Probably `$` instead of `@`, Maven complains about it, Maven specific version required?

Reply:
@nmco The [surefire](http://maven.apache.org/surefire/maven-surefire-plugin/test-mojo.html) documentation says it resolves problems with `<argLine>`, which was what was happening on my machine. I found that I had to remove the `<argLine>` entirely from the root pom.xml in order to get jacoco to run properly.  

> [INFO] --- jacoco-maven-plugin:0.8.1:report (post-unit-test) @ gs-status-monitoring ---
> [INFO] Skipping JaCoCo execution due to missing execution data file.

Did you try it with the upgraded surefire and the @{argLine} usage?